docs/lambda/configure-lambda.asciidoc

```diff
@@ -12,6 +12,7 @@ To configure APM through the AWS Management Console:
 NODE_OPTIONS                  = -r elastic-apm-node/start # use this exact fixed value
 ELASTIC_APM_LAMBDA_APM_SERVER = <YOUR-APM-SERVER-URL>     # this is your APM Server URL
 ELASTIC_APM_SECRET_TOKEN      = <YOUR-APM-SECRET-TOKEN>   # this is your APM secret token
+ELASTIC_APM_SEND_STRATEGY     = background                # recommended for steady load scenarios
```

Comment:
Is this part meant as a workaround to bridge to a possible future change in the extension to use "background" as the default without having to specify `ELASTIC_APM_SEND_STRATEGY`?